package.json

```diff
@@ -13,16 +13,16 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/action-chip": "1.0.1",
+    "@bcrs-shared-components/action-chip": "1.0.2",
     "@bcrs-shared-components/certify": "1.0.3",
-    "@bcrs-shared-components/contact-info": "1.0.6",
     "@bcrs-shared-components/confirm-dialog": "1.0.1",
+    "@bcrs-shared-components/contact-info": "1.0.8",
     "@bcrs-shared-components/detail-comment": "1.0.4",
     "@bcrs-shared-components/enums": "1.0.8",
-    "@bcrs-shared-components/fee-summary": "1.0.7",
-    "@bcrs-shared-components/interfaces": "1.0.13",
-    "@bcrs-shared-components/staff-payment": "1.0.12",
-    "@bcrs-shared-components/share-structure": "^1.0.1",
+    "@bcrs-shared-components/fee-summary": "1.0.9",
+    "@bcrs-shared-components/interfaces": "1.0.15",
+    "@bcrs-shared-components/share-structure": "1.0.3",
+    "@bcrs-shared-components/staff-payment": "1.0.14",
```

Comment:
Most of these updates it because I updated the interfaces in shared components -- but only share-structure used it -- so this should be an empty update for the other components. 🤞